addons/l10n_uk/data/account.tax.group.csv

```diff
@@ -1,5 +1,10 @@
 id,name,country_id/id
 tax_group_0,TAX 0%,base.uk
 tax_group_5,TAX 5%,base.uk
+tax_group_125,TAX 12.5%,base.uk
 tax_group_175,TAX 17.5%,base.uk
 tax_group_20,TAX 20%,base.uk
+tax_group_0RC,Reverse Charge: Customer to pay VAT to HMRC (0%),base.uk
```

Comment:
I don't think we need additional tax groups. One tax group per tax rate is enough.

Reply:
My concern for this is that fact that it's required when you're self billing that you denote the reverse charge on the invoice somewhere. 
See section (10.7) 
https://www.gov.uk/guidance/the-vat-domestic-reverse-charge-procedure-notice-735#para9